coursebuilder/modules/explorer/_static/js/course.js

```diff
@@ -15,7 +15,8 @@ $(document).ready(function documentReady() {
   });
 
   $('.course-carousel').on('changed.owl.carousel', function(event) {
-    var pos = $(this).position();
+    var pos1 = jQuery(event.target)
```

Comment:
@Sharonsyra Can we give this a more descriptive name? 

> There are only two hard things in Computer Science: cache invalidation and naming things'

Reply:
@Sharonsyra also what does `$(event.target).closest('.course-carousel')` return?

Reply:
@chegejames it works the same way. Achieves the same purpose. Curious could adding closest() be an overkill as by default event.target gets you the clicked carousel only.

Reply:
I have also tested in on browserling and it works fine for opera, firefox, safari and chrome

Reply:
@Sharonsyra Yes, it is overkill but it makes your intentions explicit as we may also want to get the position of the parent of the current .course-carousel and reposition to that.

Reply:
@chegejames I see your point. It is done.